src/storage/prisma.ts

```diff
@@ -495,21 +495,16 @@ export class PrismaStorage implements IDriveStorage {
     }
 
     async deleteDocument(drive: string, id: string) {
-        await this.db.document.delete({
-            where: {
-                id_driveId: {
+        try {
+            await this.db.document.deleteMany({
+                where: {
                     driveId: drive,
                     id: id
                 }
-            },
-            include: {
-                operations: {
-                    include: {
-                        attachments: true
-                    }
-                }
-            }
-        });
+            });
+        } catch (e) {
+            // do nothing
```

Comment:
We should ignore just that specific error, not all exceptions. Prisma usually adds an error code